examples/multi_cn/s0/local/magicdata_data_filter.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
```

Comment:
why not just use filter_scp.pl

Reply:
Just copy the data preparing script from kaldi's egs/multi_cn